mm2src/mm2_main/tests/mm2_tests/mod.rs

```diff
@@ -10,6 +10,38 @@ mod tendermint_ibc_asset_tests;
 mod tendermint_tests;
 mod z_coin_tests;
 
+mod zhtlc_native_reexport {
```

Comment:
I don't think these changes made for fixing clippy warnings? Is it?

Reply:
It is. As I remember I was making clippy working in both options with --all-features and without it

Reply:
As I understood, it was made to make `enable_z_coin` visible [here](https://github.com/KomodoPlatform/atomicDEX-API/blob/1745-clippy-all-features/mm2src/mm2_main/tests/mm2_tests/best_orders_tests.rs#L1060). However, if you make function public there is no need to re export module.

Reply:
I didn't mean only `mod zhtlc_native_reexport {` line. Sorry I should have extend the scope. I also mean `enable_z_coin` moved in this module. Don't think clippy suggests moving functions does it?

Reply:
done